packages/embed/EmbedHtmlConverter.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/*
+ * HTML converter for Paragraphs.
+ */
+module.exports = {
+
+  type: 'embed',
+  tagName: 'embed',
+
+  import: function(el, node) {
+    node.src = el.attr('src');
+    node.html = el.html();
+  },
+
+  export: function(node, el) {
+    el.attr('src', node.src)
+      .html(node.html);
+  }
+
+};
```

Comment:
A minor thing, but do if we want to be strict that filenames of classes start with a capital letter, we should call it `embedHtmlConverter.js` since we return an object here, not a full class. Or is that too picky?